400-claims.md

```diff
@@ -76,25 +76,26 @@ The CNAB claim is defined as a JSON document. The specification currently does n
+
+- `bundle` (REQUIRED): The bundle, as defined in [the Bundle Definition](101-bundle-json.md).
+- `created` (OPTIONAL): A timestamp indicating when this release claim was first created. This MUST not be changed after initial creation.
+- `custom` (OPTIONAL): A section for custom extension data applicable to a given runtime.
+- `modified` (OPTIONAL): A timestamp indicating the last time this release claim was modified. Executing the installation action MAY set this to a timestamp that matches `created`. Executing any modifying action (`upgrade`, `uninstall`, or a custom action with the property `"modifies": true`) MUST set this field to the time of the operation.
+- `name` (REQUIRED): The name of the _installation_. This can be automatically generated, though humans may need to interact with it. It MUST be unique within the installation environment, though that constraint MUST be imposed externally. Elsewhere, this field is referenced as the _installation name_. The format of this field must follow the same format used for the `name` field in the [bundle.json file specification](101-bundle-json.md#the-bundlejson-file).
+- `outputs` (OPTIONAL): Key/value pairs that were created by the operation. These are stored so that the user can access them after the operation completes. Some implementations MAY choose not to store these for security or portability reasons. See the [Outputs](#outputs) section for details. If this field is not present, it may be assumed that no outputs were generated as a result of the operation.
+- `parameters` (OPTIONAL): Key/value pairs that were passed in during the operation. These are stored so that the operation can be re-run. Some implementations MAY choose not to store these for security or portability reasons. However, there are some caveats. See the [Parameters](#parameters) section below for details.
+- `result` (REQUIRED): The outcome of the bundle's last action (e.g. if action is install, this indicates the outcome of the installation.). It is an object with the following fields:
+  - `action` (REQUIRED): Indicates the action that the current bundle is in. This may be any of the built-in actions (`install`, `upgrade`, `uninstall`) as well as any custom actions as defined in the bundle descriptor. The special name `unknown` MAY be used in the case where the CNAB Runtime cannot determine the action name of a claim.
+  - `message` (OPTIONAL): A human-readable string that communicates the outcome. Error messages MAY be included in `failure` conditions.
+  - `status` (REQUIRED): Indicates the status of the last phase transition. Valid statuses are:
+    - `failure`: failed before completion
+    - `underway`: in progress. This should only be used if the invocation container MUST exit before it can determine whether all operations are complete. Note that `underway` is a _long term status_ that indicates that the installation's final state cannot be determined by the system. For this reason, it should be avoided.
```

Comment:
I'm a little confused by 

> Note that `underway` is a _long term status_ that indicates that the installation's final state cannot be determined by the system.

Is `underway` not meant to be used to indicate that a mutation is currently being performed on the installation? I think this would be useful (and have been using a `pending` status to indicate this in internal tooling).

Reply:
I had the same expectations as @jlegrone. If tools avoid setting `underway` everytime they run a bundle  then it doesn't really address my concern from https://github.com/cnabio/cnab-spec/issues/312

Reply:
I did not change the wording of `underway` between the previous version and this one. I'm just trying to get the formatting and all fields basically covered in this PR. A follow-up issue or something is fine with me, but we're holding up a formatting PR with unrelated things.

Reply:
Renaming to `pending` is up for discussion here: https://github.com/cnabio/cnab-spec/pull/325

Reply:
@carolynvs can you follow up on #312?

Reply:
Will do!